build/Build.cs

```diff
@@ -117,19 +123,8 @@ class Build : NukeBuild
                 });
         });
 
-    Target OutputPackagesToPush => _ => _
```

Comment:
Remove output step as packages/versions can be specified/interpolated in workflow yaml file now